src/provisioner/provisioner.py

```diff
@@ -50,27 +50,36 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
             cortx_conf_url = CortxProvisioner._cortx_conf_url
         cortx_config_store = ConfigStore(cortx_conf_url)
         # source code for encrypting and storing secret key
-        cluster_id = ConfigStore.get('cluster>id')
-        if cluster_id is None:
-            raise CortxProvisionerError(errno.EINVAL, 'Cluster ID not specified')
-        encryption_key = Cipher.gen_key(cluster_id, 'cluster')
-        for key in Conf.get_keys(CortxProvisioner._solution_index):
-            # TODO: use /etc/cortx/solution/secret to confirm secret 
-            if key.endswith('secret'):
-                secret_val = Conf.get(CortxProvisioner._solution_index, key)
-                val = None
-                with open(os.path.join('/etc/cortx/solution/secret', secret_val)) as f:
-                    val = f.read()
-                if val is None:
-                    raise CortxProvisionerError(errno.EINVAL,
-                        'Could not find the Secret in /etc/cortx/solution/secret')
-                val = Cipher.encrypt(encryption_key, val.encode('ascii'))
-                Conf.set(CortxProvisioner._solution_index, key, val)
+        encryption_key = None
```

Comment:
Change variable name:
`encryption_key` to `cipher_key`

Reply:
udpated teh codeaccordingly